app/assets/stylesheets/govuk-component/_previous-and-next-navigation.scss

```diff
@@ -0,0 +1,95 @@
+.govuk-previous-and-next-navigation {
+  display: block;
+
+  @include media-down(mobile) {
```

Comment:
Should the default styling be for the narrowest viewport, then the media queries work up from there?